packages/EasyErrorHandler/src/Interfaces/ErrorHandlerInterface.php

```diff
@@ -10,6 +10,16 @@
 
 interface ErrorHandlerInterface
 {
+    /**
+     * @return \EonX\EasyErrorHandler\Interfaces\ErrorResponseBuilderInterface[]
+     */
+    public function getBuilders(): array;
```

Comment:
I don't think we should require those methods to be implemented by all ErrorHanlders when it's only a thing we need for symfony in dev/test environment.

Reply:
Yeap, agree with you. Done it with `@method` annotation in the `ErrorHandlerInterface` and moving it to the `TraceableErrorHandlerInterface`. What do you think?